src/main/java/core/basesyntax/Animal.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Animal extends Zoo implements Moveable, Feedable, Placeable {

```

Comment:
let's make this class abstract, since we don't want te create generic animal, we are interested in some specific objects like shark, eagle etc